docs/1-getting-started/06-using-features.md

```diff
@@ -38,4 +38,4 @@ For example:
 import "@ui5/webcomponents/dist/features/ColorPaletteMoreColors.js;";
 ```
 
-Next: [Typescript Support](./typescript-support)
+Next: [Typescript Support](./07-typescript-support)
```

Comment:
meanwhile there have been changes to the paths in the docs folder in the main branch, and these changes are to be compliant with the current convention to reference the file by its actual name and place (previously it was ../ and without the number in front)